JSDemos/Demos/TreeList/FormEditing/Angular/app/app.component.css

```diff
@@ -1,3 +1,3 @@
-::ng-deep #tree-list-demo {
-  min-height: 700px;
+#employees {
+  max-height: 640px;
```

Comment:
Question.

Why do we use the ```max-height``` and not just the ```height``` here?
Because before these changes we had a minimal 700px height here, but now the demo's content may be shorter.
Can it cause some height shrinking or other negative effects on the demo page?

Reply:
There are only a few demos which have `min-height`, I guess it's safe to remove it. In local demo server it's okay, but I'll check it in WG once it appears there